bigtable/bigtable-proxy/src/main/java/com/google/cloud/bigtable/examples/proxy/commands/Verify.java

```diff
@@ -0,0 +1,251 @@
+                      .setChain(
+                          Chain.newBuilder()
+                              .addFilters(RowFilter.newBuilder().setCellsPerRowLimitFilter(1))
+                              .addFilters(
+                                  RowFilter.newBuilder().setStripValueTransformer(true).build())))
+              .build();
+
+      Iterator<ReadRowsResponse> readIt =
+          stub.withDeadline(Deadline.after(1, TimeUnit.SECONDS)).readRows(readRequest);
+
+      try {
+        while (readIt.hasNext()) {
+          readIt.next();
+        }
+        System.out.println("Bigtable Read: OK");
+      } catch (StatusRuntimeException e) {
+        System.out.println("Bigtable Read: Failed - " + e.getStatus());
+        return;
+      }
+
+      CheckAndMutateRowRequest rwReq =
+          CheckAndMutateRowRequest.newBuilder()
+              .setTableName(tableName)
+              .setRowKey(ByteString.copyFromUtf8("some-non-existent-row"))
+              .setPredicateFilter(RowFilter.newBuilder().setBlockAllFilter(true))
+              .addTrueMutations(
+                  Mutation.newBuilder().setDeleteFromRow(DeleteFromRow.getDefaultInstance()))
+              .build();
+
+      try {
+        CheckAndMutateRowResponse ignored = stub.checkAndMutateRow(rwReq);
+        System.out.println("Bigtable Read/Write: OK");
+      } catch (StatusRuntimeException e) {
+        System.out.println("Bigtable Read/Write: Failed - " + e.getStatus());
+        return;
+      }
+    } finally {
+      channel.shutdown();
+    }
+  }
+
+  void checkMetrics(Credentials creds) throws IOException {
+    Instant now = Instant.now().truncatedTo(ChronoUnit.MINUTES);
+    Instant end = Instant.now().truncatedTo(ChronoUnit.MINUTES);
+
+    GCPResourceProvider resourceProvider = new GCPResourceProvider();
+    Resource resource = Resource.create(resourceProvider.getAttributes());
+    GcpResource gcpResource = ResourceTranslator.mapResource(resource);
+
+    MetricExporter exporter =
+        GoogleCloudMetricExporter.createWithConfiguration(
+            MetricConfiguration.builder()
+                .setCredentials(creds)
+                .setProjectId(metricsProjectId)
+                .setInstrumentationLibraryLabelsEnabled(false)
+                .build());
+
+    ImmutableList<MetricData> metricData =
+        ImmutableList.of(
+            ImmutableMetricData.createLongGauge(
+                resource,
+                InstrumentationScopeInfo.create("bigtable-proxy"),
+                "bigtableproxy.presence",
+                "Number of proxy processes",
+                "{process}",
+                ImmutableGaugeData.create(
+                    ImmutableList.of(
+                        ImmutableLongPointData.create(
+                            TimeUnit.MILLISECONDS.toNanos(now.toEpochMilli()),
+                            TimeUnit.MILLISECONDS.toNanos(end.toEpochMilli()),
+                            Attributes.empty(),
```

Comment:
should we tag it with "verify" or something?

Reply:
I dont think it matters...just need some sign of life